src/backend/distributed/operations/shard_split.c

```diff
@@ -358,13 +379,20 @@ ErrorIfCannotSplitShardExtended(SplitOperation splitOperation,
  * 'shardInterval'              : Source shard interval to be split.
  * 'shardSplitPointsList'		: Split Points list for the source 'shardInterval'.
  * 'nodeIdsForPlacementList'	: Placement list corresponding to split children.
+ * 'distributionColumnList'     : Maps relation IDs to distribution columns.
+ *                                If not specified, the distribution column is read
+ *                                from the metadata.
+ * 'targetColocationId'         : Specifies the colocation ID (only used for
+ *                                create_distributed_table_concurrently).
  */
 void
 SplitShard(SplitMode splitMode,
 		   SplitOperation splitOperation,
 		   uint64 shardIdToSplit,
 		   List *shardSplitPointsList,
-		   List *nodeIdsForPlacementList)
+		   List *nodeIdsForPlacementList,
+		   HTAB *partitionColumnOverrides,
```

Comment:
maybe partitionColumnOverrides -> distributionColumnOverrides

(I know, the naming is wildly inconsistent, but since the introduction of partitioning in postgres we more often refer to it as the distribution column)